### PR TITLE
chore: assert logging levels

### PR DESCRIPTION
**Pull request by rodrigo4244**

### CODE REVIEW

File: transports/helpers/assertLevel.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  'silly',
+]
+
+const assertLevel = consumerLevel => {
+  const validLevel = LOGGING_LEVELS.reduce((acc, level) => {
+    if (level === consumerLevel) return true
+    return acc
+  }, false)
+
+  if (! validLevel) throw Error(`The level ${consumerLevel} is not valid.`)
```
Comment (kewitz) [inline transports/helpers/assertLevel.js]: This message would be more meaningful if included the variable name.

File: transports/helpers/assertLevel.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  'verbose',
+  'debug',
+  'silly',
+]
+
+const assertLevel = consumerLevel => {
+  const validLevel = LOGGING_LEVELS.reduce((acc, level) => {
+    if (level === consumerLevel) return true
+    return acc
+  }, false)
```
Comment (kewitz) [inline transports/helpers/assertLevel.js]:
```js
const validLevel = LOGGING_LEVELS.reduce((acc, level) =>
  level === consumerLevel ? true : acc
, false)
```

Comment (rodrigo4244) [reply transports/helpers/assertLevel.js]: I tried this, it raises an `eslint` error -> `no-confusing-arrow`. Should I disable for this line?

**Comment by rodrigo4244** [comment]
I don't know how to add tests for it since I cannot change environmental variables on the run.
I tried to add another file and change the env vars there. What happened was that it uses the env vars from `index.spec.js` or it uses the env vars from `assertLevel.spec.js` and one of the test files fails.

File: test/helpers/assertLevel.spec.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    const msg = 'LOGGER_LEVEL'
+
+    assertLevel(level, msg)
+  })
+
+  it('should throw for an invalid logging level', () => {
+    const level = 'invalidLevel'
+    const msg = 'LOGGER_LEVEL'
+
+    assert.throws(() => assertLevel(level, msg))
```
Comment (kewitz) [inline test/helpers/assertLevel.spec.js]: :+1:

Comment (kewitz) [inline test/helpers/assertLevel.spec.js]: :+1:

File: transports/helpers/assertLevel.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  'info',
+  'verbose',
+  'debug',
+  'silly',
+]
+
+const assertLevel = (consumerLevel, msg) => {
+  const validLevel = LOGGING_LEVELS.reduce((acc, level) => (level === consumerLevel ? true : acc), false)
+
+  if (! validLevel) throw Error(`${msg} ${consumerLevel} is not valid.`)
```
Comment (kewitz) [inline transports/helpers/assertLevel.js]: Nope. The right assert behavior is
```js
throw Error(msg)
```

and then

```js
assertLevel(LOGGER_LEVEL, 'LOGGER_LEVEL invalid.')
```

**Comment by rodrigo4244** [comment]
Yep! Thanks for every tip! HAHA

**Pull request merged**